sw/legacy/common/spi.h

```diff
@@ -7,27 +7,26 @@
 
 #include "stdint.h"
 
-#define SPI_TX_REG 0
-#define SPI_STATUS_REG 4
+#define SPI_BASE 0x83000000
```

Comment:
This needs to be removed. Because we've defined this in sonata system regs header file: https://github.com/lowRISC/sonata-system/blob/main/sw/legacy/common/sonata_system_regs.h#L19-L20

Reply:
Now resolved.